src/pages/office/components/route-number/route-number.ts

```diff
@@ -46,12 +45,22 @@ export class RouteNumberComponent implements OnChanges {
 
   routeNumberChanged(routeNumber: string): void {
     if (this.formControl.valid) {
-      if (isNumber(routeNumber)) {
+      if (this.isNumeric(routeNumber)) {
         this.routeNumberChange.emit(Number.parseInt(routeNumber, 10));
       }
     }
   }
 
+  isNumeric(value: string): boolean {
```

Comment:
This would accept `01` - are we happy with that?

Reply:
for the current usages this is fine.